codalab/common.py

```diff
@@ -5,6 +5,7 @@
   - precondition, a utility method that check's a function's input preconditions.
 """
 import http.client
+from codalab.worker.file_util import BINARY_PLACEHOLDER
```

Comment:
`codalab` shouldn't depend on `codalab.worker`, so you could move `BINARY_PLACE_HOLDER` here (unless this is going to cause more complications)

Reply:
Yeah, that's true. Moved `BINARY_PLACE_HOLDER` to `codalab.common`.